lib/Driver/DarwinToolChains.cpp

```diff
@@ -492,6 +494,35 @@ toolchains::Darwin::constructInvocation(const LinkJobAction &job,
   return II;
 }
 
+
+ToolChain::InvocationInfo
+toolchains::Darwin::constructInvocation(const ArchiveJobAction &job,
+                                        const JobContext &context) const {
+   assert(context.Output.getPrimaryOutputType() == file_types::TY_Image &&
+         "Invalid linker output type.");
+
+  ArgStringList Arguments;
+
+  // Configure the toolchain.
+  const char *LibTool = "libtool";
```

Comment:
You should be able to use `ar` to do the operation.  The only time that `libtool` is really required is for fat libraries.

Reply:
I don't really have an opinion here, but from the previous review: https://github.com/apple/swift/pull/25088#discussion_r289062890

Reply:
Commented on that - I think that we can safely get away with it with `llvm-ar` at least

Reply:
Commented against it. :-(

Reply:
I think that we can get away with it using the stock Xcode tools as well (do we really care about the original AT&T Unix?).